packages/block-library/src/social-link/variations.js

```diff
@@ -304,6 +305,13 @@ const variations = [
 		title: 'WhatsApp',
 		icon: WhatsAppIcon,
 	},
+	{
+		name: 'x',
+		attributes: { service: 'x' },
+		keywords: [ 'Twitter' ],
```

Comment:
Lowercase is fine here.

Reply:
Done in a0a1b8695cf2ecb1275aabfbb7fb128522d2956d